Kitodo/src/main/java/de/sub/goobi/helper/Helper.java

```diff
@@ -493,7 +495,21 @@ public static void setCurrentUser(User user) {
      * @return metadata language as String
      */
     public static String getMetadataLanguageForCurrentUser() {
-        return (String) Helper.getManagedBeanValue("#{LoginForm.myBenutzer.metadataLanguage}");
+        return Helper.getCurrentUser().getMetadataLanguage();
+    }
+
+    /**
+     * Get AktuelleSchritteForm.
+     *
+     * @return AktuelleSchritteForm
+     */
+    public static AktuelleSchritteForm getAktuelleSchritteForm() {
```

Comment:
Please change method name as they should be in English. I know that the German name is used still as class name.

Reply:
Hmmm... What can be correct English name for it?